backend_users/tomato/rpcserver.py

```diff
@@ -53,11 +54,12 @@ def start():
 	def wrapError(error, method, args, kwargs):
 		error = handleError(error, method, args, kwargs)
 		return sslrpc2.Failure(error.raw)
-	server = sslrpc2.Server(('0.0.0.0', config.SERVER_PORT), beforeExecute=logCall, onError=wrapError, keyfile=config.SERVER_CERT,
-		certfile=config.SERVER_CERT, ca_certs=config.SERVER_CA_CERTS, cert_reqs=ssl.CERT_REQUIRED)
-	server.registerContainer(api)
-	util.start_thread(server.serve_forever)
-	print >>sys.stderr, "done."
+	for config in settings.get_own_interface_config():
```

Comment:
This is my fault since the default is multiple interfaces (eg. for backend_api, or now backend_core). The respective function in settings will only return a one-element array, so this is exactly the same as writing ```config = settings.get_own_interface_config()[0]```; but it may be able to open several ports.